provision/acc_provision/apic_provision.py

```diff
@@ -1431,6 +1431,37 @@ def cloudfoundry_tn(self):
                    ["app_profile"])
         app_epg_name = (
             self.config["cf_config"]["default_endpoint_group"]["group"])
+        custom_epg_name = (
+            self.config["aci_config"]["cloudfoundry"]["custom_epg"])
+        isolated_segment = (
+            self.config["aci_config"]["cloudfoundry"]["isolated_segment"])
+        children_epg_objs, children_apic_objects = [], []
+
+        if isolated_segment:
+            isolated_subnet = (
+            self.config["aci_config"]["cloudfoundry"]["isolated_subnet"])
+            node_subnet.append(isolated_subnet)
+
+            is_gorouter_contract = self.contract(
```

Comment:
we need a separate gorouter-contract for each isolation segment, and the so the name of the IS should be part of the contract name